Fst2List.cpp

```diff
@@ -332,34 +347,40 @@ class CFstApp {
             u_null_string), saveEntre(u_null_string), openingQuote(
             u_null_string), closingQuote(u_null_string),
 
+        inMorphoMode(false), isKorean(false), p(NULL), morphDicCnt(0),
+        
+        isMdg(false),
+        
         autoCallStack(NULL), transitionListHead(NULL), transitionListTail(NULL),
 
-        cycInfos(NULL),
-
+        cycInfos(NULL), 
         headCyc(0), cyclePathCnt(0), headCycNodes(0), cycNodeCnt(0),
-
-        inputPtrCnt(0), outputPtrCnt(0), inBufferCnt(0), outBufferCnt(0),
-
+        
+        inputPtrCnt(0), outputPtrCnt(0), 
+        inBufferCnt(0), outBufferCnt(0), 
+        alphabet(NULL), korean(NULL), processedLexicalMasks(NULL),
+        lexicalMaskCnt(0), maxLexicalMaskCnt(8),
+        
         stopSubListIdx(0) {
-    initCallIdMap();
-  }
+          initCallIdMap();
+        }
   ;
   ~CFstApp() {
     stopExploDel();
     cleanCyclePath();
     free_abstract_Fst2(a, &fst2_free);
     if (saveSep != u_null_string)
```

Comment:
For single-line statements, `if` must always have an accompanying brace.

Reply:
done